src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -74,26 +75,27 @@ class HoldRequest extends React.Component {
     });
   }
 
-  updateIsLoadingState(status) {
-    this.setState({ isLoading: status });
-  }
-
   /**
-   * requireUser()
-   * Redirects the patron to OAuth log in page if he/she hasn't been logged in yet.
+   * getNotification()
+   * Renders notification text surrounded by a 'nypl-banner-alert' toolkit wrapper.
    *
-   * @return {Boolean}
+   * @return {HTML Element}
    */
-  requireUser() {
-    if (this.state.patron && this.state.patron.id) {
-      return true;
-    }
-
-    const fullUrl = encodeURIComponent(window.location.href);
-
-    window.location.replace(`${appConfig.loginUrl}?redirect_uri=${fullUrl}`);
-
-    return false;
```

Comment:
Was this section moved up to another component; therefore, no longer required?

Reply:
Ah, it was moved down the file. ESLINT complained that the function needed to be placed after the `getNotification` function.